2021/_data/homepage.json

```diff
@@ -1,15 +1,15 @@
 {
   "og_title":"JBCNConf - The big JVM conference in Spain",
```

Comment:
```suggestion
  "og_title":"JBCNConf - The biggest JVM conference in Barcelona",
```

Reply:
Not exactly the same, plus is this kind of a slogan we have been using. I'd rather keep it as it is now